src/travis2docker/cli.py

```diff
@@ -191,7 +190,7 @@ def main(return_result=False):
     git_repo = args.git_repo_url
     git_base = GitRun.get_data_url(git_repo, False)[0]
     docker_user = args.docker_user
-    root_path = args.root_path
+    root_path = os.path.join(args.root_path, git_repo.rsplit("/", 1)[-1])
```

Comment:
This PR changes the current cache folders created

It will require a clean process for local developers and even for runbot so the impact is big

The old path used vs new one

```diff
- /Users/moylop260/.t2d/repo/git_git.vauxoo.com_vauxoo_sbd.git
+ /Users/moylop260/.t2d/sbd.git/repo/git_git.vauxoo.com_vauxoo_sbd.git 
```

In fact, it was not requested in the original issue

Could you fix the issues only with the required there with minimal changes, please?

Reply:
Done

Reply:
Thank you! 👍🏼